packages/setup-ocaml/src/cache.ts

```diff
@@ -71,13 +71,11 @@ async function composeOpamCacheKeys() {
       : // eslint-disable-next-line unicorn/no-await-expression-member
         `${platform}-${(await getSystemIdentificationInfo()).version}`;
   const architecture = getArchitecture();
-  const octokit = github.getOctokit(GITHUB_TOKEN);
-  const {
-    data: { tag_name: opamVersion },
-  } = await octokit.rest.repos.getLatestRelease({
-    owner: "ocaml",
-    repo: "opam",
-  });
+  const opamVersion =
+    platform === Platform.Win32
+      ? "0.0.0.2"
```

Comment:
Is this intended?

Reply:
This is as intended: on Windows, we continue to use the custom-built version of opam. Native support for Windows will be implemented after this PR is merged.